### ISSUE TITLE
Clippings/wx: Sanitize all strings passed to jQuery.html()

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings#179]
Clippings 6 has been rejected by a Mozilla Add-ons editor, due to not sanitizing strings passed to $(...).html().

Consider using the [DOMPurify library](https://github.com/cure53/DOMPurify/) to sanitize the strings.

**Comment by aecreations** [comment]
Also need to sanitize HTML strings used in jQuery DOM manipulation methods. For example, in new.js:
```
$('<div id="new-folder-dlg-fldr-tree" class="folder-tree"></div>').appendTo("#new-folder-dlg-fldr-tree-popup");

**Comment by aecreations** [comment]
Fixed in 6.0.3
